### PR TITLE
Fix division by 0 in arcs layer curve calculation

### PR DESCRIPTION
**Pull request by chrisdigity**
**tldr; encountered a divide by Zero condition, thought I'd share...**

Set off on a bit of a journey recently tracking down, what I thought was, a memory leak using the React Bindings repo "react-globe.gl". In my application, I am adding and removing arc layer data in real time to visualize the random network model of an active cryptocurrency network. I found that upon removing arc data, *sometimes*, I would get errors in the console:
Capture
(I can hastily reproduce this by going to <https://codesandbox.io/s/react-globe-gl-arcs-with-rings-n2mxh0?file=/src/App.tsx> and changing the timeout value on line 86 of App.tsx to a much lower number)

I thought that perhaps this error was halting a cleanup routine. Tracked down the error to an intermittent divide by Zero condition, fixed the condition, the error disappeared but the memory consumption continued.

I don't know why I didn't try it in the first place, but after trying almost everything I could think of, I switched from Firefox to Chromium and my RAMs were no longer excessively allocated to the browser.

### CODE REVIEW

**Comment by vasturiano** [comment]
@chrisdigity thanks for your PR.

Could you check if you're using the latest version of this module (`2.24.4`)?
Reason why I ask is because there's been a [recent fix](https://github.com/vasturiano/three-globe/commit/0af0b7e4a74ad56bea4843813befe07fdedaf853) which addressed this exact issue, which was occurring on the edge case of arcs with 0 length (start point = end point).

So your issue may be solved if you upgrade to the latest version.

**Comment by chrisdigity** [comment]
Ah snap, yeah this update came out between the time I encountered this and then decided to share after finishing the project. My bad.
